src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -397,7 +397,7 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 				this._updateForNotebookConfiguration();
 			}
 
-			if (e.compactView || e.focusIndicator || e.insertToolbarPosition || e.cellToolbarLocation || e.dragAndDropEnabled || e.fontSize || e.markupFontSize || e.insertToolbarAlignment) {
+			if (e.compactView || e.focusIndicator || e.insertToolbarPosition || e.cellToolbarLocation || e.dragAndDropEnabled || e.fontSize || e.outputFontSize || e.markupFontSize || e.insertToolbarAlignment || e.outputLineHeight) {
```

Comment:
fontFamily is a special case and does not trigger a style update when it's changed, so I also left outputFontFamily out of this list.
I'm not sure why it's treated as a special case though, and perhaps it can just be included in the settings object with some refactoring.

Reply:
It's an oversight, we should include that too IMHO.

Reply:
added